serverMiddleware/server-pulsar-api-proxy.js

```diff
@@ -75,6 +81,12 @@ app.all('/*', (req, res) => {
 
     reqOptions.headers['Accept'] = 'application/json'
 
+    if (ca && cert && key) {
```

Comment:
For making the connection to the Pulsar components using self-signed, only the CA is required. The cert and key are good to have since they would enable mutual TLS, but can we make it optional? `cert` and `key` need to go together, but `ca` can optionally be specified.